### ISSUE TITLE
Unbreaking allows break more stone

### ISSUE THREAD

**Issue by ImMetro** [MLG-Fortress/ExtraHardMode#213]
How do I put in the config that enchants like unbreaking can break more stone blocks than not unrbeaking. I would like to fix this please. 

because Unbreaking on an iron pick is IRON_PICKAXE#3, however IRON_PICKAXE#3@100 or something like that does not work.

**Comment by RoboMWM** [comment]
what

**Comment by RiskTolerance** [comment]
Yeah, bumping this. It would be nice to see the option of the unbreaking enchantment give more durability as it's intended.

**Comment by RoboMWM** [comment]
Other than visibility for other people who may submit PRs, if you wish to bump a request to me you can do so via [patreon](https://r.robomwm.com/patreon)

**Comment by RoboMWM** [comment]
Is this an accurate representation of why you'd like this feature @ImMetro @RiskTolerance (and @voigtman)?

As a server administrator,
I want to modify damage to tools with the "UNBREAKING" enchantment.
So that tools with the unbreaking enchantment are able to break more stone blocks than their unenchanted counterparts

**Comment by voigtman** [comment]
yes, that is correct!

**Comment by RoboMWM** [comment]
Ok thanks! Would something like this work?

Scenario: `World Rules.Mining.Inhibit Tunneling.Amount of Stone Tool Can Mine` includes the entry `IRON_PICKAXE@100`
**Given** I am a player with an iron pickaxe with the `UNBREAKING` enchantment
**When** I use the tool to mine stone
**Then** the tool's durability remains vanilla (EHM does not damage the tool)

Or would you be looking for a configuration option (multiplier? Addition? or an augmentation to the list item?)

**Comment by voigtman** [comment]
I'd want a percentage multiplier I think - I am not sure how that would look.. so Each level adds some n% to the tools life without going overboard and having to type in each possible enchantment level and it's stone breakage used on the server...

**Comment by RoboMWM** [comment]
I could do just a real number so you can use decimal values for the multiplier. I would assume it'd be a multiplier of what's already defined in the config.

Something like

Scenario: `World Rules.Mining.Inhibit Tunneling.Amount of Stone Tool Can Mine` includes the entry `IRON_PICKAXE@100`
AND `World Rules.Mining.Inhibit Tunneling.unbreaking multiplier` includes entry `IRON_PICKAXE@2.0`
Given I am a player with an iron pickaxe with the `UNBREAKING` enchantment
When I use the tool to mine stone
The tool's durability with respect to mining stone is 200 (100 * 2)

**Comment by RoboMWM** [comment]
Alright, implemented the above (the alternate scenario, e.g. IRON_PICKAXE@100@2.0). It does not account for enchantment levels, it merely checks if the enchant exists on the material and then applies the multiplier to the previously-defined number.

So followup questions are:
- Should levels be included? Or is it fine this way?
- If the multiplier is not specified e.g. `IRON_PICKAXE@100`, and the iron pickaxe has the unbreaking enchant, should tool damage still be applied, or should it be ignored entirely by EHM (100% vanilla)?

**Comment by RoboMWM** [comment]
If you'd like to test this, https://ci.appveyor.com/project/RoboMWM39862/extrahardmode/builds/36164272/artifacts
